src/NextGen/DonationForm/resources/embed.ts

```diff
@@ -0,0 +1,3 @@
+import {iframeResize} from 'iframe-resizer';
+
+iframeResize({log: true}, 'iframe[data-givewp-embed]');
```

Comment:
Should we remove the `log: true` line? That's probably for debugging?

Reply:
👍 03c0ae8c